rules.go

```diff
@@ -48,5 +48,20 @@ func (rule AllowSingleDomainAllRecrodsAllActions) is_applicable(ar Authorization
 	}
 }
 
-// TODO: add rules for manipulating particular LB
+type AllowLoadBalancersForwardingRule struct{ lb_id string }
```

Comment:
How about AllowSingleLoadBalancerAllForwardingRulesAllActions? I know it's a bit Java-esque, but IMO when it comes to permissions it's better to be explicit; same goes for `lb_id` -> `load_balancer_id`